region/lowernorfair/east.json

```diff
@@ -3503,10 +3544,167 @@
                   "notable": false,
                   "requires": [
                     "h_canNavigateHeatRooms",
+                    {"heatFrames": 600},
+                    {"enemyDamage": {
```

Comment:
I was leaving this more open to interpretation. I would assume a player would be trying to avoid them even if they have canCarefulJump turned off, and be partially successful. But I can add a second Zebbo, as that's the most likely outcome if they run through poorly.